doc/tutorials/acrn_configurator_tool.rst

```diff
@@ -18,6 +18,8 @@ Capabilities:
   XML format
```

Comment:
Would it be useful to add a note about where the board configuration file comes from to line 15?